node/src/components/consensus/protocols/highway.rs

```diff
@@ -147,7 +147,7 @@ impl<I: NodeIdT, C: Context + 'static> HighwayProtocol<I, C> {
                     highway_config.minimum_round_exponent,
                     highway_config.minimum_round_exponent,
                     highway_config.maximum_round_exponent,
-                    era_start_time,
+                    now,
```

Comment:
Shouldn't we also use `now` in the `map` above? There won't be any `prev_cp` if you restart and then reinitialize eras.

Reply:
We could, but it's less important (and maybe less correct?) in that case, because if `prev_cp` exists, that means we didn't restart, and haven't been down for a long time.

Reply:
You might be right. I thought that this `new_boxed` constructor is also used when initializing eras after the start-up, in which case there would be a `prev_cp` but that protocol instance would have just be initialized. That's incorrect though b/c we have a different era constructor there.

Reply:
Hm, actually, I'm not sure now :grimacing: I think when we're initializing eras after a restart, we still create them one by one, and they create their round success meters, so `prev_cp` would still exist even if we just restarted, wouldn't it?